freightliner_cascadia_2012_dot_10004/VehicleConfigParams.yaml

```diff
@@ -112,17 +112,14 @@ exclude_lidar: true
 excluded_lidar_topics:
   - /detection/lidar_detector/objects
   - /detection/lidar_detector/objects_markers
-  - /hardware_interface/lidar/points_raw/hardware_interface/can/brake_position
+  - /hardware_interface/lidar/points_raw
   - /hardware_interface/camera/projection_matrix/detection/lidar_detector/cloud_clusters
```

Comment:
For every freightliner config please add
/hardware_interface/velodyne_1/lidar/points_raw
/hardware_interface/velodyne_2/lidar/points_raw